app/controllers/turns_controller.rb

```diff
@@ -7,7 +8,11 @@ def create
     @conversation = Conversation.find(params[:conversation_id])
     @turn = @conversation.turns.create(turn_params)
     @conversation.turns << Turn.new(utterance: Conversation.question)
-    redirect_to conversation_path(@conversation)
+    respond_to do |format|
+      format.js {}
```

Comment:
This would look for a template create.js.erb, you made one for show.  Rails looks for a template the same name as the controller method.